### PR TITLE
Some text improvements

### PR DESCRIPTION
**Pull request by GamingDucking**
Made some grammar fixes and some improvements to readme.md.

### CODE REVIEW

File: .github/readme.md [hunk 1]
```diff
@@ -47,17 +47,17 @@ First Install The dependencies for Debian-like distributions.
...
 
-You can run emulator with Samples using this command:
+You can run the emulator with some samples using this command:
    
 `rm -f /dev/shm/rpcsx-* && ./rpcsx-os --mount  "<path to fw>/system" "/system" --mount "<path to 'game' root>" /app0 /app0/some-test-sample.elf [<args for test elf>...]`
 
 ## Creating a log
 
-You can use this flag if you get sigfault for debugging.
+You can use this flag if you got a sigfault for debugging.
```
Comment (qurious-pixel) [inline .github/readme.md]: Why only grammar, and leave spelling?
`segfault`
"got a" could be "encounter", "experience", or "receive".

Comment (GamingDucking) [reply .github/readme.md]: will fix it

File: .github/readme.md [hunk 2]
```diff
@@ -47,17 +47,17 @@ First Install The dependencies for Debian-like distributions.
...
 
-You can run emulator with Samples using this command:
+You can run the emulator with some samples using this command:
    
 `rm -f /dev/shm/rpcsx-* && ./rpcsx-os --mount  "<path to fw>/system" "/system" --mount "<path to 'game' root>" /app0 /app0/some-test-sample.elf [<args for test elf>...]`
 
 ## Creating a log
 
-You can use this flag if you get sigfault for debugging.
+You can use this flag if you encountered a sigfault for debugging purposes.
```
Comment (qurious-pixel) [inline .github/readme.md:60]: sigfault -> segfault

Comment (GamingDucking) [reply .github/readme.md:60]: didn't notice but will fix it as well

**Pull request merged**